src/components/ui/Suggest.jsx

```diff
@@ -101,6 +102,31 @@ const Suggest = ({
     }
   };
 
+  useEffect(() => {
+    if (isMobile && dropdownVisible && window.visualViewport) {
+      const TOP_BAR_HEIGHT = 60;
+      // visualViewport.height is the real visible height, not including the virtual keyboard.
+      // Giving a fixed height to the container makes the content scrollable
+      outputNode.style.height = window.visualViewport.height - TOP_BAR_HEIGHT + 'px';
+
+      const suggestions = document.querySelector('.autocomplete_suggestions');
+      const cancelTouchScrollIfNotOverflow = e => {
+        const hasOverflow =
+          suggestions &&
+          suggestions.getBoundingClientRect().height > outputNode.getBoundingClientRect().height;
+        if (!hasOverflow) {
+          e.preventDefault();
+        }
+      };
+      outputNode.addEventListener('touchmove', cancelTouchScrollIfNotOverflow);
+
+      return () => {
+        outputNode.style.height = 'auto';
+        outputNode.removeEventListener('touchmove', cancelTouchScrollIfNotOverflow);
+      };
+    }
+  }, [isMobile, items, dropdownVisible, outputNode]);
```

Comment:
Maybe this effect could also listen to [the visualViewport `resize` event](https://developer.mozilla.org/en-US/docs/Web/API/Visual_Viewport_API#interfaces), to adapt the element height after the keyboard is closed.
This is quite minor though.

<img src="https://user-images.githubusercontent.com/4726554/119960967-d04abc80-bfa5-11eb-89e3-cef4c08d7ccf.png" width="400">

Reply:
I've added it, as it was less difficult than what I thought at first.
Let's hope I didn't break anything else :P